src/pages/App.svelte

```diff
@@ -79,9 +79,7 @@
   });
 
   async function loadAuthorityBoundary(): Promise<FeatureCollection<Polygon>> {
-    const resp = await fetch(authoritiesUrl);
-    const body = await resp.text();
-    const geojson: FeatureCollection<Polygon> = JSON.parse(body);
+    let geojson = await getAuthoritiesGeoJson();
```

Comment:
While I was using that helper function, I realized it wasn't used here, but it should be